Networking-HOWTO.md

```diff
@@ -1,76 +1,74 @@
 
-Kanux supports at least the following connectivity devices:
+Kanux supports the following connectivity devices:
 
 - Ethernet
-- Kano Wireless WiFi dongle
-- Android  USB
+- WiFi
+- Android USB
 
 ### Ethernet
 
-Ethernet connectivity should always be ready functioning. Simply plug the network cable
-to a dhcp-ready network and Kanux will acquire a DHCP lease automatically.
+Ethernet connectivity should always be ready functioning. Simply plug the network cable to a dhcp-ready network and Kanux will acquire a DHCP lease automatically.
 
 ### USB devices
 
-Kanux should also play nice with Android and iPhone USB tethering devices.
-Simply plug the usb cable, enable tethering, and Kanux will acquire a DHCP lease automatically.
+In order to enable the network interface for the USB port to work with tethering you need to do the following: Edit the following file as root `sudo vi /etc/network/interfaces` and add the line `iface usb0 inet dhcp` in the end of the file, which enables the `dhcp` service for the USB port.
 
-### Wireless
+After this run `sudo ifup usb0` and reboot. Now enable tethering on your mobile device, just connect it to the Kanux and you should be online!
 
-You can setup your wireless router, or you can also put your mobile device
-in "Wireless tethering" mode. Note that Kanux fully supports the wireless dongle
-provided. Other WiFi dongles should work as well but have not been fully tested.
+### WiFi
 
-If you need to connect to other wireless networks, open the "Wifi" app icon
-in the Extras folder, or execute "sudo kano-wifi" from the command line.
+Raspbian `Jessie` is not following the usual `Debian` approach to networking with`/etc/network/if-*` scripts. Instead `dhcpcd` is used. `dhcpcd` is an implementation of the `DHCP` client specified in `RFC 2131`. It gets the host information (`IP`address, routes, etc) from a `DHCP` server and configures the network interface of the machine on which it is running.
 
-Once you successfully connect to a wireless network, it will be remembered,
-so next time you boot Kanux it will automatically connect to the network
-if it is in range.
+It then runs the configuration script which writes `DNS` information to `resolvconf`, if available, otherwise directly to `/etc/resolv.conf`. If the `hostname` is currently `blank`, (`null`) or `localhost`, or `force_hostname` is `YES` or `TRUE` or `1` then dhcpcd sets the hostname to the one supplied by the `DHCP` server.
 
-In case you need to fine tune more specific wireless secured networks,
-kano-wifi allows you to provide a custom wpa_supplicant configuration file,
-like this:
+`dhcpcd` then daemonises and waits for the lease renewal time to lapse. It will attempt to renew its lease and reconfigure if the new lease changes, when the lease begins to expire, or the `DHCP` server sends message to renew early.
 
- * sudo kano-wifi /path/to/my/wpa_supplicant.conf
```

Comment:
Might be worth mentioning the way variables leak from one hook script to the next, because they are all sourced by a single bash process.

Reply:
I don't know why that message has become attached to that line....